src/lib/dialog/README.md

```diff
@@ -40,15 +40,12 @@ export class PizzaComponent {
 
   dialogRef: MdDialogRef<PizzaDialog>;
 
-  constructor(
-    public dialog: MdDialog,
-    public viewContainerRef: ViewContainerRef) { }
+  constructor(public dialog: MdDialog) { }
 
   openDialog() {
-    let config = new MdDialogConfig();
-    config.viewContainerRef = this.viewContainerRef;
-
-    this.dialogRef = this.dialog.open(PizzaDialog, config);
+    this.dialogRef = this.dialog.open(PizzaDialog, {
+      role: 'alertdialog'
```

Comment:
What I was going for was to show that you can pass in an object, but I didn't want people copying the config over with something like `disableClose: true`. Having `disableClose: false` seemed redundant.